content/docs/studio/get-started.md

```diff
@@ -5,16 +5,10 @@ Studio].
 
 <admon>
 
-Iterative Studio [works very closely with your Git ecosystem][how-it-works]. So,
-you need to connect Iterative Studio to the Git repositories for your projects.
```

Comment:
> > added a brief about how Studio works in the get started page
>
> This should be covered in the Get Started itself, I think

And it is, in the **Sign up / sign in** section below, right?

<img width="1000" alt="image" src="https://user-images.githubusercontent.com/1477535/200647313-a8d19b22-99a2-437e-98a1-a3e511c1f588.png">

We could update that section and/or link to https://dvc.org/doc/studio/how-it-works from there but TBH I'm not seeing much info on connecting Gut repos there. Maybe to https://dvc.org/doc/studio/user-guide/prepare-your-repositories ?

Reply:
I can try to make that change.

Reply:
Updated.

Reply:
Thanks for the updates @tapadipti . I've made the intro even shorter now.

Reply:
PTAL